downloads/release-notes/1.2.10/RELEASE_NOTE.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of jBallerina 1.2.10
+
+The jBallerina 1.2.10 patch release improves upon the 1.2.9 release by introducing the features listed below and addressing a number of [bugs](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.10%22+label%3AType%2FBug+is%3Aclosed) and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.10%22+is%3Aclosed+label%3AType%2FImprovement).
```

Comment:
It seems we need to remove the phrase `introducing the features listed below` and `improvements` as we don't have any.

Reply:
Done.